ibis/backends/base/sql/alchemy/client.py

```diff
@@ -262,35 +262,8 @@ def list_schemas(self):
         )
         return self.list_databases()
 
-    def list_tables(
-        self,
-        like: Optional[str] = None,
-        database: Optional[str] = None,
-        schema: Optional[str] = None,
-    ) -> List[str]:
-        """List tables/views in the current or indicated database.
-
-        Parameters
-        ----------
-        like
-            Checks for this string contained in name
-        database
-            If not passed, uses the current database
-        schema
-            The schema namespace that tables should be listed from
-
-        Returns
-        -------
-        List[str]
-
-        """
-        inspector = self.inspector
-        # inspector returns a mutable version of its names, so make a copy.
-        names = inspector.get_table_names(schema=schema).copy()
-        names.extend(inspector.get_view_names(schema=schema))
-        if like is not None:
-            names = [x for x in names if like in x]
-        return sorted(names)
+    def list_tables(self, like=None, database=None, schema=None):
```

Comment:
this signature is completely different that the others

Reply:
This is to keep code backward-compatible, since in SQLAlchemy backends use `schema='foo'` instead of `database='foo'`. I see that to pass the test I removed the deprecation code of the `schema` argument and I haven't restored it. If you don't think it's a big deal, I'll just remove `schema`, much simpler.